deploy/chart/values.yaml

```diff
@@ -0,0 +1,77 @@
+cephfsCtrlpluginSa:
+  serviceAccount:
+    annotations: {}
+cephfsNodepluginSa:
+  serviceAccount:
+    annotations: {}
+controllerManager:
+  kubeRbacProxy:
+    args:
+    - --secure-listen-address=0.0.0.0:8443
+    - --upstream=http://127.0.0.1:8080/
+    - --v=0
+    containerSecurityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      readOnlyRootFilesystem: true
+    image:
+      repository: gcr.io/kubebuilder/kube-rbac-proxy
```

Comment:
Is this just for building, or does the helm chart install the rbac proxy container? I would expect the helm chart to only install the csi operator.

Reply:
Today csi-operator container 2 containers one is rbac proxy and another is csi-operator manager, This is auto generated files(helm charts).

Reply:
having 2 commits one for auot-generated files and one for actually changes will make easier to review wdyt?

Reply:
@subhamkrai with this everything is autogenerated except makefile.